rollup.config.js

```diff
@@ -21,7 +21,13 @@ const baseConfig = (outputFormat) => {
     input: 'src/index.js',
     plugins: [
       nodeResolve(),
-      babel(),
+      babel({
+        plugins: [
```

Comment:
Is this going to extend the existing babelrc plugins list?

Reply:
Yes, it extends so it uses .babelrc then adds accordingly.